### PR TITLE
Move CI to GitHub Actions

### PR DESCRIPTION
**Pull request by markstory**

### CODE REVIEW

File: .github/ci.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+jobs:
+  testsuite-linux:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: ['5.6', '7.2', '7.4', '8.0']
+        prefer-lowest: ['']
+        include:
+          - php-version: '7.2'
```
Comment (othercorey) [inline .github/ci.yml:21]: This should be 5.6 still, right?

**Comment by othercorey** [comment]
I think there is a github actions parse error. That usually prevents the builds. They don't log it though.

File: .github/ci.yml (hunk 1 above)
Comment (markstory) [reply .github/ci.yml:21]: Yes it should.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+          composer install --ignore-platform-reqs
+        elif ${{ matrix.prefer-lowest == 'prefer-lowest' }}; then
+          composer update --prefer-lowest --prefer-stable
+        else
+          composer install
+        fi
+
+    - name: Run PHPUnit
+      run: |
+        if [[ ${{ matrix.php-version }} == '5.6']]; then
```
Comment (othercorey) [inline .github/workflows/ci.yml:62]:
```suggestion
        if [[ ${{ matrix.php-version }} == '5.6' ]]; then
```
needs a space I believe

**Pull request merged**